packages/types/src/atoms/graphql-types.ts

```diff
@@ -1265,6 +1296,13 @@ export type EhrFlagsTodoArgs = {
 };
 
 
+/** Queries the LTHT EHR. */
+export type EhrGroupsByPatientArgs = {
```

Comment:
I had never noticed that our GraphQL query arguments make it into this repo... not entirely sure they should, given that it's open-source and meant to be uncoupled from PPM+. Issue for another time though